float32 left_pan_pwm
float32 left_tilt_pwm
float32 right_pan_pwm
float32 right_tilt_pwm
